LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -261,6 +257,16 @@ public bool EnsureConnected()
             return true;
         }
 
+        private DeviceClient CreateDeviceClient()
+        {
+            var deviceClient = DeviceClient.CreateFromConnectionString(this.connectionString, this.transportSettings);
+            deviceClient.SetRetryPolicy(new ExponentialBackoff(int.MaxValue,
```

Comment:
We realized that this retry policy makes no sense for two reasons:
1. it's equal to the default one
2. potentially specifies to retry for a huge amount of days, given the retry count set to int.maxvalue

Anyways, according to the documentation [here](https://github.com/Azure/azure-iot-sdk-csharp/blob/main/iothub/device/devdoc/retrypolicy.md) the "retry mechanism will stop after DefaultOperationTimeoutInMilliseconds which is currently set at 4 minutes"

Reply:
Absolutely - let's tackle the validation of the policy separately. This is only to have a common place for initializing it, so we need to deal with it only in one place.